### PR TITLE
feat: add query parameters to tasks contract that fulfill the pagination, filtering, and sorting epic

### PR DESCRIPTION
**Pull request by yquansah**
There has been an effort to support multiple changes on the server side for support of pagination, filtering, and sorting of tasks on the server side, to get away from doing that on the client side.

This PR adds some documentation for the parameters that were added for query on the API.

### CODE REVIEW

**Review by glinton — APPROVED** [review]

File: src/cloud/paths/tasks.yml [hunk 1]
```diff
@@ -46,6 +46,20 @@ get:
...
+        minimum: 0
+        default: 0
+      description: The number of records to skip
+    - in: query
+      name: sortBy
+      description: Field that records should be sorted by
+      schema:
+        type: string
+        enum:
+          - name
```
Comment (hoorayimhelping) [inline src/cloud/paths/tasks.yml:62]: can we mark these as not required so that the UI not sending them over won't change current behavior?

**Review by hoorayimhelping — COMMENTED** [review]
just one comment/question about these parameters being required

File: src/cloud/paths/tasks.yml (hunk 1 above)
Comment (yquansah) [reply src/cloud/paths/tasks.yml:62]: @hoorayimhelping Yes, I will do that!

**Comment by gavincabbage** [comment]
@yquansah Are we good to merge this?

**Comment by yquansah** [comment]
@gavincabbage we are good to merge this. This PR slipped through the cracks.

**Pull request merged**